tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27,py33,py34,docs
+envlist = py27,py33,py34,docs,js
```

Comment:
Can you place `js` before `docs` since it's more likely that javascript tests are more likely to fail during the build (hopefully saves a minute while watching the build output).